Assets/Scripts/TurnManager.cs

```diff
@@ -95,25 +150,28 @@ void Start()
         }
     }
 
+    void AdvanceToNextTurn()
+    {
+        moveIdx = (moveIdx + 1) % combatants.Count;
+        if (GetCurrentCombatController() != null)
+        {
+            combatCamera.GetComponent<CombatCamera>().ZoomNear(GetCurrentCombatController());
+            StartCoroutine(BeginTurnAfterDelay(0.1f));
+        }
+    }
+
     // Update is called once per frame
     void Update()
```

Comment:
Unless you have concrete plans for the Update method, it can be written as
```
void Update()
{
    if (GetCurrentCombatController() == null || !GetCurrentCombatController().isTurn)
    {
        AdvanceToNextTurn();
    }
}
```

Reply:
Seems reasonable. Done.